keyboards/handwired/dactyl_manuform/5x6_68/keymaps/default/config.h

```diff
@@ -0,0 +1,9 @@
+// Copyright 2023 Kostas Pagratis (@kpagratis)
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#pragma once
+
+
+#define MASTER_LEFT
```

Comment:
Master left isn't needed, as this is the default behavior if no other handedness is set. 

Also, split usb detect would probably be better here. 
```suggestion
#define SPLIT_USB_DETECT
```